### ISSUE TITLE
Please bring back Euro Caps large font size (or add it as an option)

### ISSUE THREAD

**Issue by sjmagy** [Marginal/EDMarketConnector#416]
Hi,
Can you please add an option that toggles the Euro Caps + Transparent window size between "small" and "large"?

I know this was just "fixed" in Issue #414 for those who need / want a "small" version but I actually preferred the slightly larger / expanded view of the transparent mode with the new Euro Caps font.  

I play at 3440x1440 so have plenty of screen real estate, and the size was literally in the perfect "Goldilocks Zone" -- it seemed to match the size of text in the notifications panel in Elite (I keep EDMC in the top right), didn't take up much more space, and was easy to read.  I LOVED that improvement!

 I know some people need / want the new "smaller" version but I find it ridiculously difficult to read in its current state.  I have to squint or shove my face right up to my screen to actually read the EDMC panel text now.  :(

**Comment by Marginal** [comment]
Both the Euro Caps and the default font are sized according to your system's text scaling setting for the monitor.

**Comment by sjmagy** [comment]
I think the Euro font is a little more difficult to read at the size where everything else is "just fine".  I know it's the price I pay for admission to the 21:9 display world (that not everything will be designed to take this into account), but I'd rather avoid changing my default system scaling just for one app.  I tried monkeying around with the Compatibility feature to change just the app's text scaling but unless I did it wrong, it didn't make a difference (probably because it's not that the text is _blurry_, it's that it's _tiny_).
